docs/guides/src/main/server/deploying_production.adoc

```diff
@@ -0,0 +1,29 @@
+<#import "/templates/guide.adoc" as tmpl>
+<#import "/templates/kc.adoc" as kc>
+<#import "/templates/options.adoc" as opts>
+<#import "/templates/link.adoc" as link>
+
+<@tmpl.guide
+title="Deploying in production"
+summary="An overview about the main aspects when deploying the server in production">
+
+The `prod` profile is activated by default whenever you run the `build` or the `start` command. You should use this profile to set any configuration option that
+makes sense when deploying the server in production.
+
+By default, the `prod` profile is very strict about the server configuration and lacks some key configuration options that are mandatory for production.
+The reason for that is that some settings are deployment-specific and must be set individually prior to deploying the server in production.
+These are the main settings that are mandatory for production:
+
+* TLS
+* Production-grade Database (e.g.: other than the default H2 database)
+* Hostname
```

Comment:
cache?

Reply:
Cache is not mandatory. We have defaults.

Reply:
In fact, I'm going to remove this guide. It is supposed to come from another PR :)

Reply:
ok